Content.Server/Items/ItemSystem.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Shared.Item;
+using Content.Shared.Verbs;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Items
+{
+    public class ItemSystem : EntitySystem
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<SharedItemComponent, GetInteractionVerbsEvent>(AddPickupVerb);
+        }
+
+        private void AddPickupVerb(EntityUid uid, SharedItemComponent component, GetInteractionVerbsEvent args)
+        {
+            if (args.Hands == null ||
+                args.Using != null ||
+                !args.CanAccess ||
+                !args.CanInteract ||
+                !component.CanPickup(args.User, popup: false))
+                return;
+
+            Verb verb = new("pickup");
+            verb.Act = () => args.Hands.TryPutInActiveHandOrAny(args.Target);
+            verb.IconTexture = "/Textures/Interface/VerbIcons/pickup.svg.192dpi.png";
+
+            // if the item already in the user's inventory (or some other storage), change the text
+            if (args.Target.IsInContainer())
```

Comment:
Yes, entities in containers are not shown on the context menu unless the user is also in the same container. However, currently you can still right-click on your hands or inventory slots directly. This bypasses the context menu and directly opens up the verb menu for the clicked entity. 

The text change was made so that you don't "pick up" and item in your pocket, or backpack, you just place it into your hands. But now I realise it also changes the text in other situations (e.g., the pickup text is changed for picking up items in a locker, while you are also in that locker, see picture). I guess I'll just change it from checking if it is in **any** container to checking if the user **IS** the container. Or just get rid of the text change, its completely unnecessary.

![Screenshot_13](https://user-images.githubusercontent.com/60421075/133037185-c31dbaa6-bbd9-4855-9370-16add529cd0b.png)

Reply:
Changed to
```cs
if (args.Target.TryGetContainer(out var container) && container.Owner == args.User)
```